Source/WebCore/platform/graphics/displaylists/DisplayListRecorder.cpp

```diff
@@ -324,6 +324,8 @@ void Recorder::beginTransparencyLayer(float opacity)
 
     appendStateChangeItemIfNecessary();
     recordBeginTransparencyLayer(opacity);
+
+    GraphicsContext::save();
```

Comment:
I think it is fine to call `GraphicsContext::save()` here since it is similar to what `GraphicsContextCG::beginTransparencyLayer()` does. But I think we should do the same thing in `BifurcatedGraphicsContext::beginTransparencyLayer()`. I think it should call `GraphicsContext::save()` also before calling `m_state.didBeginTransparencyLayer()`.

Reply:
Done.